code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/SmirnovAndrewTest.java

```diff
@@ -0,0 +1,50 @@
+package ru.unn.agile.ComplexNumber.Model;
+
+import org.junit.Test;
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class SmirnovAndrewTest {
+
+    @Test
+    public void isEmptyConstructorEqualsZeroConstructor() {
+
```

Comment:
Remove empty line, pls

Reply:
As I already mentioned in one of conversations with @Dmitry - blank lines in this case is ok

Reply:
As I already mentioned in one of conversations with @KalyanovDO - blank lines in this case is ok